src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1241,46 +1218,46 @@ static void reportReadOnlyParameterError(ParameterSymbol parameterSymbol, Syntax
 
     internal partial class RefSafetyAnalysis
     {
-        private static EscapeLevel? EscapeLevelFromScope(uint scope) => scope switch
+        private static EscapeLevel? EscapeLevelFromScope(Lifetime lifetime) => lifetime switch
```

Comment:
>EscapeLevelFromScope [](http://example.com/codeflow?start=36&length=20)

nit: doesn't have to be addressed in this PR, but we now have two names for seemingly the same thing: "Scope" and "Lifetime"

Reply:
Yes I am hoping that once we have a standalone spec for this area (think something with a similar level of precision and "self-containedness" as we have for definite assignment analysis), we will adopt the terms from it uniformly throughout.